client/src/deploy/send.rs

```diff
@@ -0,0 +1,46 @@
+use clap::{App, ArgMatches, SubCommand};
+
+use casper_node::types::Deploy;
+
+use super::creation_common;
+use crate::{command::ClientCommand, common, RpcClient};
+
+pub struct SendDeploy;
+
+// TODO: rename casper_node account type PutDeploy => SendDeploy, and account_send_deploy
+impl RpcClient for SendDeploy {
+    const RPC_METHOD: &'static str = "account_put_deploy";
+}
+
+impl<'a, 'b> ClientCommand<'a, 'b> for SendDeploy {
+    const NAME: &'static str = "send-deploy";
+    const ABOUT: &'static str =
+        "Deploy a smart contract file to the Casper network via an existing running node.";
+
+    fn build(display_order: usize) -> App<'a, 'b> {
+        let subcommand = SubCommand::with_name(Self::NAME)
+            .about(Self::ABOUT)
+            .display_order(display_order)
+            .arg(creation_common::input::arg(display_order));
+        creation_common::apply_common_creation_options(subcommand)
```

Comment:
We don't want these applied for this subcommand.  If you run `send-deploy -h` you see quite a few irrelevant options.  We only need the node_address and input options.